docs/components/arm/fake.md

```diff
@@ -4,6 +4,7 @@ linkTitle: "fake"
 weight: 34
 type: "docs"
 description: "Configure a fake arm to use for testing."
+metaDescription: "Configure a fake arm to test different models of robotic arms without any physical hardware: Navigate to the Create component menu. Enter a name for your arm, select the type arm, and select the fake model. Click Create component. Edit and fill in the attributes as applicable."
```

Comment:
If this shows up on the other end in step-wise format (like maybe [this way](https://www.google.com/search?q=how+do+i+delete+cookies+on+ie+6&source=hp&ei=kzJ6ZMerB_GfptQP6tCPuA4&iflsig=AOEireoAAAAAZHpAo41Kh7jXg-vcCihfcKYJGvUdkyMe&ved=0ahUKEwiHwdWxmaX_AhXxj4kEHWroA-cQ4dUDCAs&uact=5&oq=how+do+i+delete+cookies+on+ie+6&gs_lcp=Cgdnd3Mtd2l6EAMyBQghEKABMgUIIRCgATIFCCEQqwIyBQghEKsCMgUIIRCrAlAAWABg0gNoAHAAeACAAX6IAX6SAQMwLjGYAQCgAQKgAQE&sclient=gws-wiz)), this is fine. If this appears instead in paragraph format (like maybe [this way](https://www.google.com/search?q=what+is+a+cookie+in+a+web+browser&ei=wjJ6ZNDuM7jcptQPkvC3WA&ved=0ahUKEwjQ17bImaX_AhU4rokEHRL4DQsQ4dUDCBA&uact=5&oq=what+is+a+cookie+in+a+web+browser&gs_lcp=Cgxnd3Mtd2l6LXNlcnAQAzIECAAQRzIECAAQRzIECAAQRzIECAAQRzIECAAQRzIECAAQRzIECAAQRzIECAAQRzoKCAAQRxDWBBCwA0oECEEYAFDeBVjeBWCgB2gBcAJ4AIABAIgBAJIBAJgBAKABAcABAcgBCA&sclient=gws-wiz-serp)), I'd recommend sticking to sentence case:

```suggestion
metaDescription: "Configure a fake arm to test different models of robotic arms without any physical hardware. Navigate to the create component menu and enter a name for your arm, select the type arm, and select the fake model. Click create component. Edit and fill in the attributes as applicable."
```

Or even:

```suggestion
metaDescription: "Configure a fake arm to test different models of robotic arms without any physical hardware. Navigate to the "Create component" menu and enter a name for your arm, select the type "arm", and select the "fake" model. Click "Create component". Edit and fill in the attributes as applicable."
```

I assume we don't get access to any formatting, right?

Alternatively, consider avoiding including step-based instructions in these descriptions.